boards/esp32/MICROLITE_SPIRAM/sdkconfig.partition

```diff
@@ -0,0 +1,6 @@
+#CONFIG_ESPTOOLPY_FLASHMODE_DIO=y
+#CONFIG_ESPTOOLPY_FLASHFREQ_40M=y
+#CONFIG_ESPTOOLPY_FLASHSIZE_2MB=y
+CONFIG_PARTITION_TABLE_CUSTOM=y
+# move back up from micropython/ports/esp32 to the main project source code
+CONFIG_PARTITION_TABLE_CUSTOM_FILENAME="../../../boards/esp32/MICROLITE/custom-partitions.csv"
```

Comment:
You made a new partition table so I think you intend to point this board to the new file but here its pointed at the standard 4MB flash partition structure.